variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -169,7 +174,9 @@ def pass_aligned_filtering(left_read, right_read, counter):
     :param counter: Counter to report the number of reads filtered.
     :return: True or False
     """
-    if left_read.cigartuples[-1][1] == 'S' or right_read.cigartuples[0][1] == 'S':
+    # in CIGAR tuples the operation is coded as an integer
+    # https://pysam.readthedocs.io/en/latest/api.html#pysam.AlignedSegment.cigartuples
+    if left_read.cigartuples[-1][0] == 4 or right_read.cigartuples[0][0] == 4:
```

Comment:
```suggestion
    if left_read.cigartuples[-1][0] == pysam.CSOFT_CLIP or right_read.cigartuples[0][0] == pysam.CSOFT_CLIP:
```

Reply:
It actually took me ages to find this syntax because pysam documentation doesn't explicitly mention how this const is called...

FYI other attributes from the [list](https://pysam.readthedocs.io/en/latest/api.html#pysam.AlignedSegment.get_cigar_stats) are also available by stripping the `BAM_` prefix, e.g. `BAM_CEQUAL` becomes `pysam.CEQUAL`